ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -137,12 +131,36 @@ class AlertsView extends React.Component {
           this.getParamsFromFilters(updatedFilters),
         );
       }
-      this.setState({ filters: currentFilters }, this.fetchAlertSummaries);
+      this.setState(
+        {
+          filters: currentFilters,
+          notSupportedAlertFilters: this.selectNotSupportedFilters(
+            currentFilters.filterText,
+          ),
+        },
+        this.fetchAlertSummaries,
```

Comment:
What if someone was only filtering by one of the unsupportedFilters? That should probably be checked before `this.fetchAlertSummaries` is called.

Reply:
Because `this.fetchAlertSummaries()` is called in `componentDidMount()`, we can assume that the unsuported filters are already available.
Just tested and it works, loading the page for the first time and entering an unsupported filter.

Even so, you are right. 
From a theoretical, logical point of view, it's better to `fetchAlertSummaries` first and after this to `selectNotSupportedFilters`.
It's not OK to base our code on something which happens in another function.
To be sure that we already `fetchAlertSummaries`, I think it's best to `await this.setState(
        { filters: currentFilters },
        this.fetchAlertSummaries,
      );`, and after this to `selectNotSupportedFilters`

Reply:
Maybe I'm misunderstanding something, but what I was trying to say is that if the only filters someone has entered are unsupported filters, we probably don't want to call `this.fetchAlertSummaries`. Just throw the error about the unsupported filters and don't call the function at all.

Reply:
Since the work is almost done and there isn't too much to do, I think that @airimovici or @ionutgoldan can work further on this.

It shouldn't take more than 2 hours, as my gut suggests.